projects/libaom/project.yaml

```diff
@@ -7,6 +7,8 @@ sanitizers:
 - undefined
 auto_ccs:
 - jhuoponen@google.com
+- jianj@google.com
```

Comment:
James: Let's also delete jhuoponen because he's no longer the PM.

Roshan says he doesn't need to be cc'ed automatically.

I think we should change the primary_contact to Jingning (or you, or me).

Reply:
Done.